developer/src/kmc-model/build.sh

```diff
@@ -13,7 +13,7 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 builder_describe "Keyman kmc Lexical Model Compiler module" \
   "@/common/web/keyman-version" \
   "@/developer/src/common/web/test-helpers" \
-  "@/common/models/templates test" \
+  "@/common/models/templates" \
```

Comment:
In package.json, this now needs to be a dependency, not a devDependency.

Reply:
This is marked as resolved but models-templates doesn't appear to be a non-dev dependency yet in package.json?

Reply:
Oh, right, not sure why that part didn't click for me before.  Fixed now.